decent-dao/fractal-interface
# Update |  Safe Proposal Details

## Description
This PR adds and updates components as needed to implement Multisig Safe Transaction/Proposal Details. 


## Notes



## Issue / Notion doc (if applicable)
Closes #749


## Testing
To do some testing you are going to want a DAO with at least 2 signers (since creating a proposal does the first sign)
- Create Proposal
- Switch to 2nd signer account
- Approve/Sign Tx
- Execute Tx

You should see:
- Wonderful data populate and update as you do these actions
- Useful Toast during signing/execution
- Not being able to click on things when you are not suppose to (if you are not an owner or during the transaction execution)


## Screenshots (if applicable)
![localhost_3000_(Desktop) (19)](https://user-images.githubusercontent.com/38386583/206886954-d67be40f-8230-474b-aa4f-309209af9aba.png)

![localhost_3000_(Desktop) (18)](https://user-images.githubusercontent.com/38386583/206886958-38aac077-58bc-4dcd-a9c4-2c8d1b482309.png)

![image](https://user-images.githubusercontent.com/38386583/206886940-23ff67ec-fc37-4746-b8c7-2a37dc66af80.png)

![localhost_3000_(Desktop) (17)](https://user-images.githubusercontent.com/38386583/206886960-22027f98-b96a-48c6-ba8b-84faa6072f22.png)

![image](https://user-images.githubusercontent.com/38386583/206886971-40c554f7-05ef-4641-b043-e575e40f9a32.png)

![localhost_3000_(Desktop) (20)](https://user-images.githubusercontent.com/38386583/206887044-509e6e34-5847-467f-b04c-ca864268ec24.png)

![localhost_3000_ (7)](https://user-images.githubusercontent.com/38386583/206887934-007508f6-0e5a-4ea6-a33d-38fb150b2d7b.png)
